common.yml

```diff
@@ -673,6 +695,11 @@ post:
           set -v
         rm -rf /data/db/*
         exit 0
+  # Extra post steps needed for mongodump passthrough.
+  - func: f_resmoke_report_attach
+  - func: f_gotest_report
```

Comment:
Is this necessary for the resmoke stuff? Did you just copy this from the mongosync config?

Reply:
It's copied from the mongosync config without any deeper analysis.